src/components/Profile.jsx

```diff
@@ -16,5 +17,11 @@ class Profile extends React.Component {
 		);
 	}
 }
-
+Profile.propTypes = {
+	user: PropTypes.shape({
+		images: PropTypes.array.isRequired,
+		firstName: PropTypes.string.isRequired,
+		lastName: PropTypes.string.isRequired,
+	}),
```

Comment:
You marked all of the fields as isRequired, but you actually want user to be required. You can't access `this.props.user.firstName` is `user` is undefined.

Leave images as is though, making it required is good.